chall03/gbudau.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+import sys
+import curses.ascii
```

Comment:
I like this. I think for this specific case `curses.ascii.isdigit()` > regular `.isdigit()`